app/components/primer/popover_component.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module Primer
+  class PopoverComponent < Primer::Component
```

Comment:
maybe we should add a `render?` definition, checking the necessary slots

Reply:
Yeah! I feel like the message slot should be required at least.